scripts/validate_releases.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Used for local testing to make sure that correct Xcode version is used.
+# sudo xcode-select --switch /Applications/Xcode_13.2.1.app/Contents/Developer/
+# XCODEBUILD=/Applications/Xcode_13.2.1.app/Contents/Developer/usr/bin/xcodebuild
+XCODEBUILD=xcodebuild
+
+CURRENT_DIRECTORY=$(pwd)
+CURRENT_XCODEBUILD_DIRECTORY=$(xcode-select -print-path)
+echo "Current xcodebuild path: ${CURRENT_XCODEBUILD_DIRECTORY}."
+
+cd Tests/SPMTest/UISPMTest/
+
+# Dictionary that stores Navigation SDK version and expected xcodebuild result after resolving dependencies.
+# xcodebuild exits with codes defined by sysexits(3): https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/sysexits.3.html.
+# It will exit with:
+# - EX_OK on success.
+# On failure, it will commonly exit with: 
+# - EX_USAGE if any options appear malformed.
+# - EX_NOINPUT if any input files cannot be found.
+# - EX_IOERR if any files cannot be read or written.
+# - EX_SOFTWARE if the commands given to xcodebuild fail.
+# It may exit with other codes in less common scenarios.
+declare -A EXPECTED_XCODEBUILD_RESULTS
+EXPECTED_XCODEBUILD_RESULTS=( \
+    [2.4.0]=74 \
+    [2.4.1]=74 \
+    [2.4.2]=0 \
+    [2.5.0]=74 \
+    [2.5.1]=74 \
+    [2.5.2]=74 \
+    [2.5.3]=0 \
+    [2.6.0]=0 \
+    [2.7.0]=0 \
+    [2.7.1]=0 \
```

Comment:
`74` essentially means that SPM failed to resolve dependencies for specific Navigation SDK version. Tests are performed on Xcode `v13.2.1`.